docs/modules/ROOT/pages/dataformats.adoc

```diff
@@ -4,15 +4,25 @@
 
 ## Generic Read and Write Functions
 
-The `Formats` library contains `read`, and `write` functions for input and output of other data formats, e.g. XML or CSV. The XML and CSV plugins are built into the standard DataSonnet distributions while other formats can be supported by implementing `DataFormatPlugin` interface.
+The `Formats` library contains `read`, and `write` functions for parsng and generating data in other data formats, e.g. XML or CSV. The XML and CSV plugins are built into the standard DataSonnet distributions while other formats can be supported by implementing `DataFormatPlugin` interface.
```

Comment:
minor typo, "parsng". Also, what do you think about "serializing" and "deserializing"? I think those are the most standard terms for what's happening here